docs/troubleshooting.md

```diff
@@ -70,4 +70,4 @@ Grammars will only be updated in patch releases if the patch release is specific
 
 Note: New languages will not appear in GitHub's search results for some time after the pull request has been merged and the new Linguist release deployed to GitHub.com.
 This is because GitHub's search uses [go-enry](https://github.com/go-enry/go-enry) for language detection which tends to lag behind Linguist by a few weeks to months.
-This in turn requires an update to the underlying search code once go-enry is inline with Linguist.
+This in turn requires an update to the underlying search code once go-enry is inline with Linguist (See also go-enry's list of [divergences from Linguist](https://github.com/go-enry/go-enry#divergences-from-linguist)).
```

Comment:
I'm not sure this is relevant enough to Linguist to warrant inundating readers with the gory details of portability differences between various regular expression engines. Keep in mind, most users perusing this document are probably in a rush, so they're more likely to expect missing details to be included in pages linked to from a troubleshooting answer.

Reply:
Maybe there is too many elements in that list that won't be relevant to most people,  but I think there should be at least a mention that go-enry doesn't support `linguist-language` overrides in the `.gitattrubutes`.

I know that I was confused about this multiple time until I discovered that in the list of divergences.

It's especially weird when you click on a language featured in the language stats bar and code search tells you there is no files with that language in the project :/

Reply:
> It's especially weird when you click on a language featured in the language stats bar and code search tells you there is no files with that language in the project :/

Wait, so *that's* the culprit behind all those flaky, confusing code-listings seen when clicking certain languages in project stat-bars? 😕 Finally, I now know where years of [“missing” INI files](https://github.com/search?q=repo%3AAlhadis%2F.files++language%3AINI&type=code) went in my dotfiles repository…

> but I think there should be at least a mention that go-enry doesn't support `linguist-language` overrides in the `.gitattributes`

This would be more suited to a separate troubleshooting entry explaining why some files aren't being listed when filtering by language (ideally after the [_“My repository is detected as the wrong language”_](https://github.com/github/linguist/blob/9b53b4b0450194fb7990bda3823d982ccb7bd9b3/docs/troubleshooting.md#my-repository-is-detected-as-the-wrong-language) entry). Such an addition warrants its own PR, however.

In the mean time, we should avoid repeating ourselves; this troubleshooting entry already includes a link to the [`go-enry`](https://github.com/go-enry/go-enry) repository; users can find their own way to the [_“Divergences from Linguist”_](https://github.com/go-enry/go-enry#divergences-from-linguist) section if they so choose to.